sdks/python/apache_beam/internal/apiclient.py

```diff
@@ -24,15 +24,13 @@
 import logging
 import os
 import re
-import time
-
 from StringIO import StringIO
-
-from apitools.base.py import encoding
-from apitools.base.py import exceptions
+import time
 
 from apache_beam import utils
 from apache_beam.internal.auth import get_service_credentials
+from apache_beam.internal.clients import storage
+import apache_beam.internal.clients.dataflow as dataflow
```

Comment:
Could this be `from apache_beam.internal.clients import dataflow` to match the other imports.